.github/workflows/realm.yml

```diff
@@ -3,7 +3,7 @@ name: realm
 on:
   pull_request:
     paths:
-      - "realm/**"
+      - "r/**"
```

Comment:
please rebase and also fix for `.github/workflows/deploy-realm.yml`

Reply:
Done -> Can you please verify the changes? -> https://github.com/gnolang/gnochess/pull/171/commits/0f83b2d043c1e12e69cacde9f3d54f8aac73aee7